Python/Twitter_Scrapper/twitter_scrapper.py

```diff
@@ -0,0 +1,303 @@
+"""
+This script extracts all Tweets, Retweets, Images, Videos, Hashtags,
+Likes and Pinned tweet of a specific user in a .csv file.
+"""
+
+import csv
+import random
+import time
+from itertools import zip_longest
+
+import tweepy
+from bs4 import BeautifulSoup
+from selenium import webdriver
+
+# Authenticate to Twitter
+CONSUMER_KEY = "<your-consumer-or-API-key-goes-here>"
+CONSUMER_SECRET = "<your-consumer-or-API-secret-goes-here>"
+ACCESS_KEY = "<your-access-key-goes-here>"
+ACESS_SECRET = "<your-access-secret-goes-here>"
```

Comment:
Can you please add all of this in a `.env` file and use a module like `decouple` to import them in the Code? Also please don't push the `.env` into the PR and instead put a `.env.example` just for the other Devs to understand on what values do they need to fill.

Reply:
Add the file .env.example